domain.yml

```diff
@@ -157,35 +161,71 @@ responses:
     - title: No, cancel the transaction
       payload: /deny
   utter_ask_transfer_form_confirm:
-  - text: 'Would you like to transfer {currency}{amount_of_money} to {PERSON}?'
-    buttons:
-    - title: Yes
-      payload: /affirm
-    - title: No, cancel the transaction
-      payload: /deny
+  - buttons:
```

Comment:
Yaml change is confusing here, the previous formatting is more consistent with the docs so that should be reverted.

Reply:
ugh so sorry, I started Rasa X in localmode and it shuffled everything 😒